app/js/components/manageApps/ManageApps.jsx

```diff
@@ -15,6 +15,7 @@ import BreadCrumbComponent from '../breadCrumb/BreadCrumbComponent.jsx';
 import { ApiHelper } from '../../helpers/apiHelper';
 import { AddonList } from './AddonList.jsx';
 import DeleteAddonModal from './DeleteAddonModal.jsx';
+import InvalidZipUploadModal from './InvalidZipUploadModal';
```

Comment:
Please add the file extension to this import statement, to ensure consistency across import style in the codebase.